src/main/java/org/jabref/logic/ai/AiDefaultPreferences.java

```diff
@@ -0,0 +1,17 @@
+package org.jabref.logic.ai;
+
+import org.jabref.preferences.AiPreferences;
+
+public class AiDefaultPreferences {
+    public static final boolean ENABLE_CHAT = false;
+    public static final boolean CUSTOMIZE_SETTINGS = false;
+    public static final AiPreferences.ChatModel CHAT_MODEL = AiPreferences.ChatModel.GPT_3_5_TURBO;
+    public static final AiPreferences.EmbeddingModel EMBEDDING_MODEL = AiPreferences.EmbeddingModel.ALL_MINILM_L6_V2;
+    public static final String SYSTEM_MESSAGE = "You are an AI assistant that analyses research papers.";
+    public static final double TEMPERATURE = 0.7;
+    public static final int MESSAGE_WINDOW_SIZE = 1_000; // "2 + 2 = 4, 3 + 3 = 6" has size 20. Source: {@link dev.langchain4j.memory.chat.TokenWindowChatMemoryTest#should_evict_multiple_orphan_ToolExecutionResultMessages_when_evicting_AiMessage_with_ToolExecutionRequests_when_SystemMessage_is_present}
```

Comment:
Double check: We need these settings also when having openai doint the injection?

Reply:
> "2 + 2 = 4, 3 + 3 = 6"

Did I really wrote that comment?

Also, it's wrong, I switched from message window to token window. I need to change this